src/IssueViz.Security/Commands/ShowToolWindowCommand.cs

```diff
@@ -70,11 +70,11 @@ public static async Task InitializeAsync(AsyncPackage package)
 
             try
             {
-                toolWindowService.Show(HotspotsToolWindow.ToolWindowId);
+                toolWindowService.Show(toolWindowId);
             }
             catch (Exception ex) when (!ErrorHandler.IsCriticalException(ex))
             {
-                logger.WriteLine(string.Format(Resources.ERR_HotspotsToolWindow_Exception, ex));
+                logger.WriteLine(string.Format(Resources.ERR_ShowToolWindow_Exception, ex));
```

Comment:
I think it's worth printing the toolWindowId or the commandId, anything that will help us identify which window was it that failed.